rover_msgs/GPS.lcm

```diff
@@ -6,5 +6,14 @@ struct GPS {
     int32_t longitude_deg;
     double longitude_min;
     double bearing_deg;
+    // Units: degrees from true
+    // 999: Invalid/Nonexistent measurement
```

Comment:
If we are going to make individual fields take up multiple lines (including comments) can you put a newline between them? It's not clear to me what comments go to what field in all cases.

Reply:
good call